consensus/vbft/service.go

```diff
@@ -199,8 +206,62 @@ func (self *Server) Halt() error {
 	return nil
 }
 
+func (self *Server) StartEmergency(height uint32) {
```

Comment:
this whole function requires metaLock.Lock(), necessary? 
seems only 'self.emergency = true'